_drafts/2017-06-07-checkers.md

```diff
@@ -0,0 +1,115 @@
+ 
+It turns out there's a *lot* to do here, and a lot of interest! We put up a proposal as a [GitHub issue]; before the unconf even began, the thread had over 50 posts and the discussion had yielded two design documents led by Hadley Wickham on [research compendium organization] and [automated build systems]. There were easily four or five projects wrapped up in the proposal.
+ 
+The thread also revealed many schools of thought. As [Nick] put it, "The problem with [opinionated analysis development] is that everyone has an opinion." We never reached consensus on basic issues like directory structure and build systems.
+
+![](http://imgur.com/RpjyfL8.jpeg)
+ 
+In the face of a beast of a topic, and so many unresolved decisions, we decided to tackle a modest slice - a guide for compendium review, and a first pass at a package for automated checks of various best practices.
+
+# Reviewing a research compendium
+
+How does one do peer review for a research compendium in the face of so much heterogeneity in types and styles of analysis?  We spent the better part of our first day brainstorming everything that one might review, from the meta-topics ("Is there a clear question?") to  minutiae ("Are `library()` calls at the top of the script?"), and mapping these out on an ever more data-dense whiteboard:
+
+![Nick, Jennifer, Molly and Alice discussing analysis best practices](http://imgur.com/vb4E3JV.jpeg)
+ _Nick, Jennifer, Molly and Alice discussing analysis best practices_
+ 
+![The team prioritizing data analysis workflow best practice into tiers](https://i.imgur.com/dIT1sjK.jpg)
+ _The team prioritizing data analysis workflow best practice into tiers_
+ 
+![With our 7th team member, Wy T. Board](http://imgur.com/8klgK7Q.jpeg)
+_With our 7th team member, Wy T. Board_
+
+These review topics made their way into a [Google doc] which will form the basis of a review guide along the lines of rOpenSci's [package review guides](https://github.com/ropensci/onboarding/#-useful-documents-in-this-repository).  One major organizing dimension we hit upon was "Tiers" - as the number of best practices can be overwhelming, it is better to prioritize them so that users have a way of advancing through escalating levels of detail.  Another was "automatability," which is key to the other half of our work.
+
+![](https://i.imgur.com/lWpcEfb.png)
+
+An example of Automatable & "Nice to have" check of best practice looks like this.:-
+
+    Research phase : Package/Organisational
```

Comment:
I don't understand how/where `checkers` does this. I'm naive, but do think this could be clarified, specifically for this box.